apps/prairielearn/elements/pl-units-input/units_utils_test.py

```diff
@@ -33,22 +33,34 @@ def test_only_units_grading_fn(
 
 
 @pytest.mark.parametrize(
-    "a_true, a_sub, atol, expected_grade",
+    "a_true, a_sub, rtol, atol, expected_grade",
     [
-        ("1m", "1 meter", "1cm", True),
-        ("1m", "1.009 meter", "1cm", True),
-        ("1m", "3.28 feet", "1cm", True),
-        ("1m", "1.010001 meters", "1cm", False),
-        ("1m", "1 foot", "1cm", False),
-        ("1m", "1 m/s", "1cm", False),
-        ("1m", "1 second", "1cm", False),
-        ("1m", "1 us", "1cm", False),
+        ("1m", "1 meter", 0.0, "1cm", True),
+        ("1m", "1.009 meter", 0.0, "1cm", True),
+        ("1m", "3.28 feet", 0.0, "1cm", True),
+        ("1m", "1.010001 meters", 0.02, "0m", True),
+        ("1m", "1.2 meters", 0.1, "0.1m", True),
```

Comment:
Why is this True? Both the `rtol` and `atol` should give us valid submission ranges of 0.9m to 1.1m? Same question for the first kilofoot test below.

Reply:
The rtol and atol are applied together, so atol contributes 0.1 and rtol 0.1 * 1 = 0.1 for a total of 0.2. If you look at the grading function, it just does unit conversions and uses the same comparison function as pl-number-input, so I think this is identical behavior to that element.

Reply:
Oh! Is this a change in `pl-number-input`? It should be `rtol` OR `atol`, not a sum of them.

Reply:
Oh no, this has always been the pl-number-input behavior. We're just mimicking that here.

Reply:
Oh right, we are using `np.allclose()` which adds them. No worries, all good.